### PR TITLE
Update md tag name

### PR DESCRIPTION
**Pull request by VipulMascarenhas**

### CODE REVIEW

File: ads/aqua/deployment.py [hunk 1]
```diff
@@ -122,14 +121,13 @@ def from_oci_model_deployment(
         aqua_service_model_tag = freeform_tags.get(
             Tags.AQUA_SERVICE_MODEL_TAG.value, None
         )
+        aqua_model_name = freeform_tags.get(Tags.AQUA_MODEL_NAME_TAG.value, None)
```
Comment (lu-ohai) [inline ads/aqua/deployment.py:124]: Remember to be told to use empty string instead of `None`

File: ads/aqua/deployment.py [hunk 2]
```diff
@@ -264,6 +254,8 @@ def create(
             if tag in aqua_model.freeform_tags:
                 tags[tag] = aqua_model.freeform_tags[tag]
 
+        tags.update({Tags.AQUA_MODEL_NAME_TAG: aqua_model.display_name})
```
Comment (lu-ohai) [inline ads/aqua/deployment.py:257]: ```tags.update({Tags.AQUA_MODEL_NAME_TAG.value: aqua_model.display_name})```

**Review by lu-ohai — CHANGES_REQUESTED** [review]

File: ads/aqua/deployment.py (hunk 1 above)
Comment (VipulMascarenhas) [reply ads/aqua/deployment.py]: ah yes. updated.

File: ads/aqua/deployment.py (hunk 2 above)
Comment (VipulMascarenhas) [reply ads/aqua/deployment.py]: yep, realized while testing. fixed.

**Review by lu-ohai — APPROVED** [review]

**Pull request merged**